scripts/top-search-queries/get-top-search-results.go

```diff
@@ -178,39 +175,22 @@ func callSearchAPI(ctx context.Context, query, nlpWeighting, apiURL string, numR
 // readQueriesFromFile opens the text file named "search-queries.txt", in the local directory, and reads it into a
 // []string object, which it returns.
 func readQueriesFromFile(ctx context.Context, cfg runConfig) (listOfQueries []string, err error) {
-	logData := log.Data{"File name: ": cfg.InputFileName}
+	logData := log.Data{"file_name": cfg.InputFileName}
 	file, err := os.Open(cfg.InputFileName)
 	if err != nil {
-		log.Fatal(ctx, "failed opening file", err, logData)
+		log.Fatal(ctx, "failed opening input file", err, logData)
 	}
-	bufferedReader := bufio.NewReader(file)
-
-	var queryString string
-	for {
-		line, err := bufferedReader.ReadString('\n')
-		if err != nil {
-			if err == io.EOF {
-				// add query from last line, of input file, unless it's empty
-				queryString = strings.TrimSpace(line)
-				if queryString == "" {
-					log.Info(ctx, "ignoring last line of input file as it's empty")
-					break
-				}
-				listOfQueries = append(listOfQueries, queryString)
-				break
-			}
-			log.Fatal(ctx, "error while reading file - failed reading row of text", err, logData)
-			break
-		}
-		// add query to list
-		queryString = strings.TrimSpace(line)
-		listOfQueries = append(listOfQueries, queryString)
+	defer file.Close()
```

Comment:
GoLand complains about an unhandled error here

Reply:
Fixed it